optuna/storages/rdb/storage.py

```diff
@@ -63,8 +63,10 @@ def create_new_study_id(self, study_name=None):
                                   direction=structs.StudyDirection.NOT_SET)
         session.add(study)
         if not self._commit_with_integrity_check(session):
-            raise ValueError(
-                "study_name {} already exists. Please use a different name.".format(study_name))
+            raise structs.DuplicatedStudyError(
+                "Another study with name '{}' already exists. "
+                "Please specify a different name, or reuse the existing one "
+                "by setting `load_if_exists` or `--skip-if-exists` flag.".format(study_name))
```

Comment:
It'd be better to describe that `load_if_exists` is for Python and that `--skip-if-exists` is for CLI. 

```
"Please specify a different name, or reuse the existing one "
"by setting `load_if_exists` (for Python API) or `--skip-if-exists` flag (for CLI).".format(study_name))
```